tests/helpers/index.test.ts

```diff
@@ -6,7 +6,7 @@ describe('misc', () => {
     const snippet = {
       content: 'const memoizedSyntaxes = useMemo()',
       created_at: '2019-09-29T19:23:31',
-      id: 364066,
+      id: '364066',
```

Comment:
[snippet type](https://github.com/xsnippet/xsnippet-web/blob/master/src/store/index.ts#L7) describes this prop as a string